bitcoind-rpc/src/main/scala/org/bitcoins/rpc/config/BitcoindInstance.scala

```diff
@@ -107,16 +117,23 @@ object BitcoindInstance extends InstanceFactory[BitcoindInstance] {
                          proxyParams = proxyParams)
   }
 
-  lazy val DEFAULT_BITCOIND_LOCATION: File = {
+  val bitcoindLocationFromConfigFile: File = {
```

Comment:
this should be a `lazy val` or `def` otherwise the program could fail there is no config file or `bitcoin-s.bitcoind-rpc.binary` option in the config file